cli/src/test/java/com/devonfw/tools/ide/io/IdeProgressBarTestImpl.java

```diff
@@ -34,27 +34,34 @@ public class IdeProgressBarTestImpl implements IdeProgressBar {
    * @param max maximum length of the bar.
    */
   public IdeProgressBarTestImpl(String name, long max) {
-
+    super(max);
     this.start = Instant.now();
     this.name = name;
     this.max = max;
```

Comment:
Why are we making another local field max, instead of using maxLength or AbstractIdeProgressBar? the super call seems completely unnecessary when not using maxLength.

Reply:
Thanks I've removed the duplicated field and added a getter for maxLength now.

Reply:
Resolved